tests/resources/functions/swift/index.swift

```diff
@@ -12,6 +12,8 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
         "APPWRITE_FUNCTION_USER_ID": req.variables["APPWRITE_FUNCTION_USER_ID"],
         "APPWRITE_FUNCTION_JWT": req.variables["APPWRITE_FUNCTION_JWT"],
         "APPWRITE_FUNCTION_PROJECT_ID": req.variables["APPWRITE_FUNCTION_PROJECT_ID"],
+        "APPWRITE_FUNCTION_EXECUTION_ID": req.variables["APPWRITE_FUNCTION_EXECUTION_ID"],
+        "APPWRITE_FUNCTION_CLIENT_IP": req.variables["APPWRITE_FUNCTION_CLIENT_IP"],
```

Comment:
Amazing job updating tests too 🔥
Let's also update test themselves in `FunctionsCustomClientTest.php` and `FunctionsCustomServerTest.php`. You can look for tests for `APPWRITE_FUNCTION_USER_ID`, and in those places, add test for `APPWRITE_FUNCTION_EXECUTION_ID` and `APPWRITE_FUNCTION_CLIENT_IP`.